ariadne/cardano/config/configuration.yaml

```diff
@@ -14837,7 +14874,7 @@ testnet_wallet: &testnet_wallet
     applicationVersion: 0
     lastKnownBlockVersion:
       bvMajor: 0
-      bvMinor: 1
+      bvMinor: 0
```

Comment:
Block version is downgraded?

Reply:
Well, that's `testnet_wallet`, so I suppose it's ok. Probably it was a mistake before.
Anyway, I took this file from `cardano-sl` as is.